commcare_connect/templates/account/base.tailwind.html

```diff
@@ -0,0 +1,57 @@
+{% load static i18n %}
```

Comment:
Can we delete the `account/base.html`?

Reply:
I think that is still referenced on some of the pages. I have only changed the base template for pages that are part of the login signup onboarding flow.

Reply:
Will those change eventually? It feels odd that previously pages that shared a base template no longer will

Reply:
Yeah, I will add changes for those pages. There are some `account` pages for a logged in user that are not a part of the onboarding flow, so I will add changes for those soon.